src/_gettsim/arbeitslosengeld_2/einkommen.py

```diff
@@ -98,24 +97,23 @@ def bruttoeinkommen_m(
     Note: Since 2023, Arbeitslosengeld 2 is referred to as Bürgergeld.
     """
     return (
-        einkommensteuer__einkünfte__aus_nichtselbstständiger_arbeit__bruttolohn_m
+        einnahmen__aus_nichtselbstständiger_arbeit__bruttolohn_m
         + einkommensteuer__einkünfte__aus_selbstständiger_arbeit__betrag_m
         + einkommensteuer__einkünfte__aus_vermietung_und_verpachtung__betrag_m
-        + einkommensteuer__einkünfte__aus_kapitalvermögen__kapitalerträge_m
-        + einkommensteuer__einkünfte__sonstige__alle_weiteren_m
-        + einkommensteuer__einkünfte__sonstige__rente__sonstige_private_vorsorge_m
-        + einkommensteuer__einkünfte__sonstige__rente__geförderte_private_vorsorge_m
-        + einkommensteuer__einkünfte__sonstige__rente__betriebliche_altersvorsorge_m
-        + sozialversicherung__rente__altersrente__betrag_m
-        + sozialversicherung__rente__erwerbsminderung__betrag_m
+        + einnahmen__aus_kapitalvermögen__kapitalerträge_m
+        + einnahmen__renten__sonstige_private_vorsorge_m
+        + einnahmen__renten__geförderte_private_vorsorge_m
+        + einnahmen__renten__betriebliche_altersvorsorge_m
+        + einnahmen__renten__gesetzliche_m
+        + einnahmen__sonstige__alle_weiteren_m
```

Comment:
```suggestion
        + einnahmen__sonstige_alle_weiteren_m
```
If we are getting rid of the `sonstige` namespace for `renten` (which is good, I think), we should do so all the way.